DragaliaAPI.Database/ApiContext.cs

```diff
@@ -67,6 +67,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         modelBuilder.Entity<DbQuest>().HasKey(e => new { e.DeviceAccountId, e.QuestId });
 
+        modelBuilder
+            .Entity<DbAbilityCrestSet>()
+            .HasKey(key => new { key.DeviceAccountId, key.AbilityCrestSetNo });
+
```

Comment:
you can use [PrimaryKey(nameof(DeviceAccountId), nameof(AbilityCrestSetNo))] on the db class instead